---
layout: post
title: "Structuromics-Part 1"
description: ""
author_handle: richa
category: blog
tags: [journal club, Structuromics, omics, Mass spectrometry,limited proteolysis, LiP-MS, structural barcodes]
---
*Research Paper Highlight:*
 **Dynamic 3D proteomes reveal protein functional alterations at high resolution in situ**

Valentina Cappelletti, Thomas Hauser, Ilaria Piazza, Monika Pepelnjak, Liliana Malinovska, Tobias Fuhrer, Yaozong Li, Christian Dorig, Paul Boersema, Ludovic Gillet, Jan Grossbach, Aurelien Dugourd, Julio Saez-Rodriguez, Andreas Beyer, Nicola Zamboni, Amedeo Caflisch, Natalie de Souza, and Paola Picotti.

[Cell 184, 545â€“559, January 21, (2021); DOI: 10.1016/j.cell.2020.12.021](https://pubmed.ncbi.nlm.nih.gov/33357446/)


In many ways the tale of blind men and the elephant is a good analogy for our understanding of biological systems. We create tools to probe biomolecules and make interpretations from our observations. But no one experimental design or approach is bias-free. To truly understand function, we need multi-dimensional, holistic analyses from complementary observational viewpoints. Technological improvements have allowed merger of biological disciplines that tradionally used non-overlapping approaches to address the same biological questions. One such case that strikes me as spectacular is what has been termed as 'structuromics'-- a combination of genomic, proteomic, genetic and cell biological approaches to ascertain protein structure and function in the cellular context. There have been multiple really cool papers published in the last few months that I'd like to highlight in a multi-part structuromics series. 

Part 1 of the series is a paper that combines proteomics and structural biology by employing limited proteolysis and mass spectrometry to predict large scale, and biologically relevant, structural and conformational changes in proteins in response to stress.  
<!--more-->

See here for [Part 2]({% post_url 2021-02-20-GI-mapping-protein-structure-paper %}). 

In this work, the authors employ 'limited proteolysis coupled to mass spectrometry', or LiP-MS, to obtain information about structural changes in thousands of cellular proteins simultaneously, in response to various stresses. The idea is a simple one - if you subject a protein to proteases, the exposed parts of the protein are readily digested, while the structured/ modified/ complexed part that are hidden are protected. Now, if the protein undergoes structural changes upon binding to a ligand or other proteins, the acessibility and the pattern of protease digestion of the protein will change. The experiment, then, can be repeated under many different conditions, and for many different proteins by employing mass spectrometry to generate quantitative profiles or fingerprints of structural alterations. 

Biochemists have used limited proteolysis as a tool to study individual proteins for a long time. What is remarkable about the LiP-MS and other similar approaches is the wealth of information on thousands of cellular proteins that can be obtained at the same time, allowing to see higher order patterns in cellular pathways, which is not possible when studying one protein at a time. Additionally, many proteins aren't as amenable to pure structural analyses, and these approaches can infer structural information that can guide protein functional studies. By applying their approach to E. coli as well as yeast cells, the authors show that monitoring conformational changes in different conditions provides important functional information that is otherwise missed when looking at protein abundance measurements alone. 

The approach is useful on its own, but its richness can be greatly enhanced if it can be combined with complementary workflows. For instance, combining it with other mass spec based readouts- protein abundance, phosphorylation, ubiquitylation, and other modifications, cross-linking based peptide analysis, etc. can tell what mechanisms underlie the conformational changes. Similarly, combining with evolutionary analyses to look at co-evolution rates of amino acid residues within a protein or interacting partners can predict the conservation mechanisms underlying the structural changes. Proteins with low resolution crystal or cryo-EM structures too can be significantly enriched with the knowledge of known conformational changes. There is a lot of room for development of modifications to the approach, and all that can be done is truly exciting!



